app/views/posts/show.html.erb

```diff
@@ -1,3 +1,30 @@
-<%= link_to users_path(1) do%>
-<h1>A specific post from a specific user</h1>
-<% end %>
\ No newline at end of file
+  <% content_for :stylesheets do %>
+  <%= stylesheet_link_tag "post" %>
+<% end %>
+
+<div class="User-Text">
+  <div class="title-and-counters">
+    <div class="tittle-users">
+      <h1 class="post-txt"><%= @post.title %></h1>
+      <p class="post-txt">by: <%= @post.user.name %></p>
+    </div>
+    <div  class="counters">
+      <p class="post-txt">Likes: <%= @post.likes_counter %></p>
+      <p class="post-txt">Comments: <%= @post.comments_counter %></p> 
+    </div>
+  </div>
+    <p class="post-content"><%= @post.text %></p>
+</div>
+<div class="comments">
+  <% if @post.comments.present? %>
+    <% @post.comments.each do |comment| %>
+      <p class="txt"><strong class="txt"><%= comment.user.name %>:</strong> <%= comment.content %></p>
+    <% end %>
+  <% else %>
+    <p class="txt">There are no comments.</p>
+  <% end %>
+</div>
+
+    <div class="button-container">
+    <button type="button" class="button-posts"><%= link_to "Pagination", users_path %></button>
+    </div>
```

Comment:
- [ ] Great job on advancing your project's UI to align with the provided wireframe! 👍 However, I noticed a minor detail that could enhance the user experience and ensure a seamless match with the design specifications.

### 👉 **Recommendation:**
  - Consider relocating the `Pagination` button to `http://localhost:3000/users/:under_id/posts` instead of its current location at `http://localhost:3000/users/:user_id/posts/:post_id`, aligning precisely with the wireframe outlined in the project requirements. See below for more details:
  
![blog_user_all_posts](https://github.com/CarlosZ96/Blog-App-MCV/assets/107192076/80fa3ed0-53e6-44e5-bdff-0ffcb234fb60)


  - This adjustment will not only ensure visual consistency but also contribute to an improved user experience, aligning your implementation closely with the intended design.

Awesome progress so far! Let's keep refining those details to create a stellar user interface. 
**Happy coding!** 🚀

Reply:
![image](https://github.com/CarlosZ96/Blog-App-MCV/assets/125598459/7f0ee3fa-8f54-4463-9491-aeaba25ef9e7)
Done!